src/main/kotlin/org/antlr/v4/runtime/PlSqlLexerBase.kt

```diff
@@ -0,0 +1,10 @@
+package org.antlr.v4.runtime
+
+abstract class PlSqlLexerBase(input: CharStream?) : Lexer(input) {
```

Comment:
Why do need this class?

Reply:
Example of grammar contains this class. I can correct grammar to exclude it in future